opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -267,6 +267,10 @@ class TraceOptions(int):
     def get_default(cls) -> "TraceOptions":
         return cls(cls.DEFAULT)
 
+    @property
+    def recorded(self) -> bool:
```

Comment:
To avoid confusion and incompatibility with the current spec version, I'd suggest renaming this and the TraceOptions flag to SAMPLED (https://www.w3.org/TR/trace-context/#trace-flags). The recorded property would be the current is_recording_events of the span.

Reply:
That sounds fine to me, changed in b7f1324.